requirements/requirements-cv.txt

```diff
@@ -1,5 +1,5 @@
 imageio>=2.5.0
-opencv-python-headless>=3.4.5.20
+opencv-python-headless>=4.1.1.26
```

Comment:
why? 🤔

Reply:
because of requirements of albumentations >= 0.5.1 (version of which I forgot to set)